src/main/java/frc/robot/commands/CmdArmSetShoulderAngle.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.commands;
+
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.subsystems.Arm;
+
+public class CmdArmSetShoulderAngle extends InstantCommand
+{
+    private final Arm            _armSubsystem;
+    private final Double _shoulderAngle;
```

Comment:
Fix the whitespace on line 9, and use `double` instead of `Double` for line 10 (and also for the parameter in the constructor.